resolver_winrt.c

```diff
@@ -409,17 +409,18 @@ bool proxy_resolver_winrt_global_cleanup(void) {
     return true;
 }
 
-proxy_resolver_i_s *proxy_resolver_winrt_get_interface(void) {
-    static proxy_resolver_i_s proxy_resolver_winrt_i = {proxy_resolver_winrt_get_proxies_for_url,
-                                                        proxy_resolver_winrt_get_list,
-                                                        proxy_resolver_winrt_get_error,
-                                                        proxy_resolver_winrt_wait,
-                                                        proxy_resolver_winrt_cancel,
-                                                        proxy_resolver_winrt_create,
-                                                        proxy_resolver_winrt_delete,
-                                                        true,  // get_proxies_for_url is handled asynchronously
-                                                        true,  // get_proxies_for_url takes into account system config
-                                                        proxy_resolver_winrt_global_init,
-                                                        proxy_resolver_winrt_global_cleanup};
+const proxy_resolver_i_s *proxy_resolver_winrt_get_interface(void) {
+    static const proxy_resolver_i_s proxy_resolver_winrt_i = {
+        proxy_resolver_winrt_get_proxies_for_url,
+        proxy_resolver_winrt_get_list,
+        proxy_resolver_winrt_get_error,
+        proxy_resolver_winrt_wait,
+        proxy_resolver_winrt_cancel,
+        proxy_resolver_winrt_create,
+        proxy_resolver_winrt_delete,
+        true,   // get_proxies_for_url is handled asynchronously
+        false,  // get_proxies_for_url does not take into account system config
```

Comment:
This `true` to `false` change.

Reply:
Can you split the true/false change into a separate commit for this project?